src/open_inwoner/scss/components/Header/AnchorMenu.scss

```diff
@@ -82,6 +82,30 @@
     }
   }
 
+  // nested list
+  &__sublist {
+    list-style-type: '- ';
+    padding-left: var(--spacing-large);
+    padding-top: var(--spacing-large);
+    line-height: var(--spacing-large);
+
+    li {
+      border-left: none;
+    }
+    li:not(:last-child) {
+      margin-bottom: var(--spacing-large);
+    }
+    li > a {
+      font-size: small;
```

Comment:
Please use var(--font-size-body-small) for this - becasue 13px is too small here.